Deprecated react removal (#45426)
mui/material-ui: examples/material-ui-vite-tailwind-ts/README.md

> # Material UI - Vite example in TypeScript

Reviewer: ```suggestion
# Material UI - Vite example with Tailwind CSS v3 in TypeScript
```

<!-- #target-branch-reference -->

```bash

> curl https://codeload.github.com/mui/material-ui/tar.gz/master | tar -xz --strip=2 material-ui-master/examples/material-ui-vite-ts
> cd material-ui-vite-ts

Reviewer: Isn't this example with tailwind css?
```suggestion
curl https://codeload.github.com/mui/material-ui/tar.gz/master | tar -xz --strip=2 material-ui-master/examples/material-ui-vite-tailwind-ts
cd material-ui-vite-tailwind-ts
```

```

Install it and run:

```bash
npm install
npm run dev
```

or:

<!-- #target-branch-reference -->

> [![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/sandbox/github/mui/material-ui/tree/master/examples/material-ui-vite-ts)
> 
> [![Edit on StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/mui/material-ui/tree/master/examples/material-ui-vite-ts)

Reviewer: ```suggestion
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/sandbox/github/mui/material-ui/tree/master/examples/material-ui-vite-taiwind-ts)

[![Edit on StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/mui/material-ui/tree/master/examples/material-ui-vite-tailwind-ts)
```

## The idea behind the example

> This example uses [Vite.js](https://github.com/vitejs/vite).

Reviewer: ```suggestion
This example demonstrates how you can use [Tailwind CSS](https://tailwindcss.com/) and [Vite](https://github.com/vitejs/vite) together with Material UI.
```
